client/source/components/RR/RatingBreakdown.jsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const RatingBreakdown = () => {
+
+}
+
+export default RatingBreakdown;
```

Comment:
August mention it is best practice to include one blank line at the end of every script

Reply:
Added these in my latest push!